.gitignore

```diff
@@ -1,12 +1,24 @@
+# Ignores files in the main git directory unless explicitly mentioned.
+# This so that files/archives created/added by the external files archive will not automatically be added,
+# as well as tmp files that might be generated by unit testing (the current MRUnit temporary fix creates
+# symlinks in the working directory to simulate distributed cache) and could appear in git when using it
+# at the same time when unit tests are ran.
+/*
+!/.gitignore
+!/README.md
+!/ArchiveGenerator.sh
+!/external_files.txt
+!/*/
+
 # External files archive info files that were added by extracting the archive over the project folder
-archive_files.txt
-archive_files_info.md
+#archive_files.txt
+#archive_files_info.md
```

Comment:
Noted. Were initially left in as reminder, but should indeed be removed for the pull request.